atom/browser/default_app/main.js

```diff
@@ -51,12 +51,12 @@ if (option.file && !option.webdriver) {
     // Run the app.
     require('module')._load(packagePath, module, true);
   } catch(e) {
+    console.error('App threw an error when running', e);
```

Comment:
why move this , since you have already handled `MODULE_NOT_FOUND` in dialog ? besides we are not throwing if its `MODULE_NOT_FOUND` .